source/Octopus.Cli/Commands/Releases/PreventReleaseProgressionCommand.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Threading.Tasks;
+using Octopus.Cli.Infrastructure;
+using Octopus.Cli.Repositories;
+using Octopus.Cli.Util;
+using Octopus.Client;
+using Octopus.Client.Exceptions;
+using Octopus.Client.Model;
+
+namespace Octopus.Cli.Commands.Releases
+{
+    [Command("prevent-release-progression", Description = "Prevent a release from progressing to next phase.")]
+    public class PreventReleaseProgressionCommand : ApiCommand, ISupportFormattedOutput
+    {
+        ProjectResource project;
+        ReleaseResource release;
+
+        public PreventReleaseProgressionCommand(IOctopusClientFactory clientFactory, IOctopusAsyncRepositoryFactory repositoryFactory, IOctopusFileSystem fileSystem, ICommandOutputProvider commandOutputProvider)
+            : base(clientFactory, repositoryFactory, fileSystem, commandOutputProvider)
+        {
+            var options = Options.For("Preventing release progression.");
+            options.Add("project=", "Name or ID of the project", v => ProjectNameOrId = v);
+            options.Add("version=|releaseNumber=", "Release number to use for auto deployments.",
```

Comment:
Why explicitly call out auto deploments?

Reply:
ah my bad, forgot to change this description.